src/Altinn.Notifications.Integrations/PlatformSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Altinn.Notifications.Integrations
+{
+    /// <summary>
+    /// Represents settings needed for communication with the Profile components.
+    /// </summary>
+    public class PlatformSettings
+    {
+        /// <summary>
+        /// Gets or sets the Profile component API base url.
+        /// </summary>
+        public string ProfileEndpointAddress { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the name of the subscription key request header.
+        /// </summary>
+        public string ProfileSubscriptionKeyHeaderName { get; set; } = "Ocp-Apim-Subscription-Key";
```

Comment:
Could we give this property a more generic name? I guess we will need the same header for requests towards register shortly.

Reply:
It's technically possible for API Management to have a different header name for each API. Though I don't see why we would ever use that I kind of like having support for it in the client.